pkg/controllers/monitoring-stack/controller.go

```diff
@@ -39,6 +39,12 @@ import (
 
 	"github.com/go-logr/logr"
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// Status condition type available
+	available = "Available"
```

Comment:
this need to be a type instead of raw string so that it can methods can accept and return condition type instead of a raw string.

Reply:
Outdated